<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="bgwidget" native="true">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>751</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#bgwidget{background-color:qlineargradient(spread:reflect, x1:0.631, y1:0.840909, x2:0.266591, y2:0.13, stop:0 rgba(232, 255, 255, 255), stop:1 rgba(230, 147, 230, 255));}
</string>
   </property>
   <widget class="QPushButton" name="boton_editar_cuenta">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>460</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius:20px;
font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(170, 255, 255);
	color: rgb(0, 85, 255);
}
QPushButton:hover{
background-color: rgb(200, 255, 255);
	color: rgb(0, 170, 255);
}</string>
    </property>
    <property name="text">
     <string>Editar cuenta</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:28pt;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Gestiona tu cuenta</string>
    </property>
   </widget>
   <widget class="QLabel" name="adv_ingreso">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>440</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>751</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255, 255, 255,200);
font: 48pt &quot;Century Gothic&quot;;
color: rgb(255, 130, 253);</string>
    </property>
    <property name="text">
     <string>MIR</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_regreso">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>15</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-image: url(&quot;return.png&quot;);
background-repeat: no-repeat;
background-position: center;
background-color: rgb(255, 255, 255);
}
QPushButton:hover{
background-color:rgb(255, 220, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_cerrar_sesion">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>15</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-image: url(&quot;signout.png&quot;);
background-repeat: no-repeat;
background-position: center;
background-color: rgb(255, 255, 255);
}
QPushButton:hover{
background-color:rgb(255, 220, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>155</y>
      <width>476</width>
      <height>286</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255,50);
border:2px solid rgb(255, 130, 253);
border-radius:20px;</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>455</width>
       <height>449</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255,50);
border:0px solid rgb(255, 85, 255);
border-radius:20px;</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_13">
        <property name="styleSheet">
         <string notr="true">QLabel{
background-color: rgba(255, 255, 255, 0);
border:10px solid;
border-color: rgba(255, 255, 255, 0);
font-size:15pt;
color: rgb(255, 130, 253);
}</string>
        </property>
        <property name="text">
         <string>Cuenta</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="styleSheet">
           <string notr="true">QLabel{
background-color: rgba(255, 255, 255, 0);
border-color: rgba(255, 255, 255, 0);
font-size:12pt;
}</string>
          </property>
          <property name="text">
           <string>Correo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="correo_label">
          <property name="styleSheet">
           <string notr="true">QLabel{
background: rgba(255, 255, 255, 50);
border: 2px solid rgb(170, 0, 255);
border-radius:10px;
font-size:12pt;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="styleSheet">
           <string notr="true">QLabel{
background-color: rgba(255, 255, 255, 0);
border-color: rgba(255, 255, 255, 0);
font-size:12pt;
}</string>
          </property>
          <property name="text">
           <string>Contrase√±a</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="contra_label">
          <property name="styleSheet">
           <string notr="true">QLabel{
background: rgba(255, 255, 255, 50);
border: 2px solid rgb(170, 0, 255);
border-radius:10px;
font-size:12pt;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(254, 142, 252);
border:20px;</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="styleSheet">
         <string notr="true">QLabel{
background-color: rgba(255, 255, 255, 0);
border:10px solid;
border-color: rgba(255, 255, 255, 0);
font-size:15pt;
color: rgb(255, 130, 253);
}</string>
        </property>
        <property name="text">
         <string>Datos usuario</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">QLabel{
background-color: rgba(255, 255, 255, 0);
border-color: rgba(255, 255, 255, 0);
font-size:12pt;
}</string>
          </property>
          <property name="text">
           <string>Nombres</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="nombres_label">
          <property name="styleSheet">
           <string notr="true">QLabel{
background: rgba(255, 255, 255, 50);
border: 2px solid rgb(170, 0, 255);
border-radius:10px;
font-size:12pt;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">QLabel{
background-color: rgba(255, 255, 255, 0);
border-color: rgba(255, 255, 255, 0);
font-size:12pt;
}</string>
          </property>
          <property name="text">
           <string>Apellidos</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="apellidos_label">
          <property name="styleSheet">
           <string notr="true">QLabel{
background: rgba(255, 255, 255, 50);
border: 2px solid rgb(170, 0, 255);
border-radius:10px;
font-size:12pt;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="styleSheet">
           <string notr="true">QLabel{
background-color: rgba(255, 255, 255, 0);
border-color: rgba(255, 255, 255, 0);
font-size:12pt;
}</string>
          </property>
          <property name="text">
           <string>Usuario</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="nickname_label">
          <property name="styleSheet">
           <string notr="true">QLabel{
background: rgba(255, 255, 255, 50);
border: 2px solid rgb(170, 0, 255);
border-radius:10px;
font-size:12pt;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="styleSheet">
           <string notr="true">QLabel{
background-color: rgba(255, 255, 255, 0);
border-color: rgba(255, 255, 255, 0);
font-size:12pt;
}</string>
          </property>
          <property name="text">
           <string>Fecha de nacimiento</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="fecha_nacimiento_label">
          <property name="styleSheet">
           <string notr="true">QLabel{
background: rgba(255, 255, 255, 50);
border: 2px solid rgb(170, 0, 255);
border-radius:10px;
font-size:12pt;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="boton_editar_datos">
    <property name="geometry">
     <rect>
      <x>415</x>
      <y>460</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius:20px;
font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(170, 255, 255);
	color: rgb(0, 85, 255);
}
QPushButton:hover{
background-color: rgb(200, 255, 255);
	color: rgb(0, 170, 255);
}</string>
    </property>
    <property name="text">
     <string>Editar datos</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>5</number>
  </property>
  <property name="gridDeltaY">
   <number>5</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
